src/battle-tooltips.ts

```diff
@@ -651,9 +651,9 @@ class BattleTooltips {
 			if (clientPokemon) {
 				if (pokemon.status === 'tox') {
 					if (pokemon.ability === 'Poison Heal' || pokemon.ability === 'Magic Guard') {
-						text += ' <small>Would take if ability removed: ' + Math.floor(100 / 16) * Math.min(clientPokemon.statusData.toxicTurns + 1, 15) + '%</small>';
+						text += ' <small>Would take if ability removed: ' + Math.floor(100 / 16 * Math.min(clientPokemon.statusData.toxicTurns + 1, 15)) + '%</small>';
```

Comment:
I think you can add `const damage = Math.floor(100 / 16 * Math.min(clientPokemon.statusData.toxicTurns + 1, 15))` after line 652 and use the variable in both places.